subworkflows/functional_annotation/README.md

```diff
@@ -65,7 +48,8 @@ These command line tool parameters can be changed by overriding the `ext.args` v
 ```nextflow
 process {
     withName: 'INTERPROSCAN' {
-        ext.args = '-f TSV --iprlookup --goterms -pa -dp -t p'
+        //
```

Comment:
What is the comment here?

Reply:
I need to add this 
// -dp is always set by default and -f is now controlled via value(out_ext)

Reply:
```suggestion
```